src/features/parsers/parseMidi.ts

```diff
@@ -29,7 +29,8 @@ export default function parseMidi(midiData: ArrayBufferLike): Song {
         i,
         {
           name: track.name,
-          instrument: track.instrument.name,
+          // infer percussion soundfont for drums
+          instrument: (track?.instrument?.family === 'drums' ? 'percussion' : track.instrument.name),
```

Comment:
Looking at the MIDI Spec, (and our dependency on tonejs/midi),  I think a more correct thing to is to check for the channel. Seems like channel 10 is guaranteed to correspond to percussion. See https://github.com/Tonejs/Midi/blob/1d3a1f539a2de55a448687285a393bc3aff7da1a/src/InstrumentMaps.ts#L151-L161

Additionally, we should probably load a different soundfont for each of the patches: https://github.com/Tonejs/Midi/blob/master/src/InstrumentMaps.ts#L151-L161

Reply:
If you look at [GM MIDI ](https://en.wikipedia.org/wiki/General_MIDI) page, they claim its channel 10. I wouldn't be surprised if zero-indexing makes it channel 9

Reply:
Ah! So it's a MIDI vs. musician mismatch. MIDI is 0-indexed, whereas vernacular is 1-indexed for channels. Checking for `9` is correct

Reply:
With that said, do you still prefer that I check the channel here, or family would work?

Reply:
channel. family is dependent on the parser we use. channel feels more correct.